src/main/java/com/iexec/worker/executor/TaskManagerService.java

```diff
@@ -277,7 +279,7 @@ boolean reveal(String chainTaskId, TaskNotificationExtra extra) {
         return isValidChainReceipt(chainTaskId, oChainReceipt);
     }
 
-    ReplicateDetails uploadResult(String chainTaskId) {
+    ReplicateStatusDetails uploadResult(String chainTaskId) {
```

Comment:
I don't think the uploadResult(.) method should return a ReplicateStatusDetails
Maybe something like returning a UploadResultDetails which will be converted to a ReplicateStatusDetails just before updating the status is better?

Reply:
Yes, I'm planning to do that in the next PR.